connector/third_party/apache-spark/src/main/java/com/google/cloud/spark/bigquery/ArrowSchemaConverter.java

```diff
@@ -468,11 +468,21 @@ final long getLong(int rowId) {
     @Override
     final UTF8String getUTF8String(int rowId) {
       long epoch = accessor.get(rowId);
+      long seconds = epoch / 1000000;
```

Comment:
Please convert to the magic numbers to constants (`ONE_MILLION`, `ONE_BILLION`)

Reply:
Done